src/Makefile

```
@@ -0,0 +1,14 @@
+ 
+a:
+	@python3 Version1.py 
+
+b:
```

es recomendable usar nombres más claros para las reglas del makefile

Tal vez podrían usar algo como:
```suggestion
test_version1:
	@python3 Version1.py 

version2:
```

 Dejo ejemplos de dos alternativas, aunque pueden usar la que quieran. Lo importante sería como lo "explican" en su README. POdrían incluir algo como:

Para ejecutar un ejemplo, se pueden usar los siguientes comandos:
````
cd src
make a
````

Justo para que los comandos hagan más sentido, por eso son los comentarios anteriores. Si sacan el makefile, y cambian el nombre de la regla, podrían poner en su README algo como:

Para ejecutar un ejemplo, se pueden usar los siguientes comandos:
````
make test_version1
````